### PR TITLE
feat: rename payment pointer to wallet address

### LINKED ISSUE #286: payment pointers -> wallet addresses
Replace all mentions of 'payment pointers' with 'wallet addresses'.

### PR DESCRIPTION
**Pull request by njlie**
## Changes proposed in this pull request

Replaces all mentions of 'payment pointer' with 'wallet address'.

## Context

Fixes #286.

### CODE REVIEW

File: openapi/resource-server.yaml
Comment (BlairCurrey) [inline openapi/resource-server.yaml:1]: I think we need to bump this (and `auth-server.yaml`). Seems like we've been doing minors bumps on the spec version (such as the last change to the resource server spec here) https://github.com/interledger/open-payments/commit/dd193c7ea6b62fde977a5ea336771cb7afb09dde#diff-9debc2038853698222051b2c2a428cee93e9dda6af6076de559971933879580d

**Review by BlairCurrey — COMMENTED** [review]
I think we need to bump the specs that changed too but looks good other than that

**Comment by BlairCurrey** [comment]
looks like there will be stuff to rename in https://github.com/interledger/open-payments/pull/289/files

File: packages/open-payments/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This package exports two clients, an `UnauthenticatedClient` and an `Authenticat
 ### `UnauthenticatedClient`
 
 This client allows making requests to access publicly available resources, without needing authentication.
-The three available resources are [Payment Pointers](https://docs.openpayments.guide/reference/get-payment-pointer), [Payment Pointer Keys](https://docs.openpayments.guide/reference/get-payment-pointer-keys), and [ILP Stream Connections](https://docs.openpayments.guide/reference/get-ilp-stream-connection).
+The three available resources are [Wallet Addresses](https://docs.openpayments.guide/reference/get-payment-pointer), [Wallet Address Keys](https://docs.openpayments.guide/reference/get-payment-pointer-keys), and [ILP Stream Connections](https://docs.openpayments.guide/reference/get-ilp-stream-connection).
```
Comment (mkurapov) [inline packages/open-payments/README.md:37]:
```suggestion
The three available resources are [Wallet Addresses](https://docs.openpayments.guide/reference/get-wallet-address), [Wallet Address Keys](https://docs.openpayments.guide/reference/get-wallet-address-keys), and [ILP Stream Connections](https://docs.openpayments.guide/reference/get-ilp-stream-connection).
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The code for the landing [page](https://openpayments.guide) is in `./landing`.
 
 ## Dependencies
 
-- [Payment Pointers](https://paymentpointers.org)
+- [Wallet Addresses](https://paymentpointers.org)
```
[the remark this answers is not in the document]
Comment (njlie) [reply README.md:30]: I'm content to remove it. I wasn't sure if we were going to update it or not to reflect wallet addresses but if not I don't see a need to maintain a reference to payment pointers.

**Review by BlairCurrey — APPROVED** [review]

**Pull request merged**